org.eclipse.sisu.plexus/src/main/java/org/codehaus/plexus/PlexusTestCase.java

```diff
@@ -20,11 +20,10 @@
 import org.codehaus.plexus.component.repository.exception.ComponentLookupException;
 import org.codehaus.plexus.context.Context;
 import org.codehaus.plexus.context.DefaultContext;
-
-import junit.framework.TestCase;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 
 public abstract class PlexusTestCase
```

Comment:
Not sure about this change. This is part of what we export and it is still based on JUnit 3! I would rather deprecate it, leave it in place as is and establish a new plexus junit5 extension.